tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -635,4 +651,30 @@ function() {
 		$tag = wp_get_attachment_image( $attachment_id, 'medium', false, array( 'class' => "wp-image-{$attachment_id}" ) );
 		$this->assertNotSame( $tag, webp_uploads_img_tag_update_mime_type( $tag, 'the_content', $attachment_id ) );
 	}
+
+	/**
+	 * Tests whether additional mime types generated only for allowed image sizes or not.
+	 *
+	 * @test
+	 */
+	public function it_should_create_mime_types_for_allowed_sizes_only() {
+		add_filter(
+			'webp_uploads_image_sizes_with_additional_mime_type_support',
+			function( $sizes ) {
+				$sizes['allowed_size_400x300'] = 'allowed_size_400x300';
```

Comment:
Let's use the new format here:
```suggestion
				$sizes['allowed_size_400x300'] = true;
```

Reply:
Good catch. Updated.